src/test/java/module/series/SeriesFixturesTest.java

```diff
@@ -0,0 +1,90 @@
+package module.series;
+
+import core.file.xml.TeamStats;
+import core.model.series.Paarung;
+import core.util.HODateTime;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class SeriesFixturesTest {
```

Comment:
I suggest naming the class for testing for the class it is testing:
`Spielplan`.

Reply:
Renamed both to MatchFixtures(Test)